llvm/lib/Transforms/Vectorize/VPlanRecipes.cpp

```diff
@@ -1231,6 +1231,8 @@ void VPWidenCallRecipe::execute(VPTransformState &State) {
 
   CallInst *V = State.Builder.CreateCall(Variant, Args, OpBundles);
   setFlags(V);
+  if (CallingConv::ID ID = Variant->getCallingConv())
+    V->setCallingConv(ID);
```

Comment:
I'm not sure why IRBuilder doesn't do this itself but I see this idiom used all over the place so I'll not worry about it.

That said, looking at existing uses can `Function::getCallingConv()` fail?  Other instanced just do:
```
CallInst *V = State.Builder.CreateCall(Variant, Args, OpBundles);
V->setCallingConv(Variant->getCallingConv());
```

FYI: GitHub will not let me attach this to the relevant line but for information I note just above there is the assert
```
assert(Variant != nullptr && "Can't create vector function.");
```
and yet on entry to the function we've already done
```
FunctionType *VFTy = Variant->getFunctionType();
```